rest/src/main/java/eu/openg/guessnumberapi/rest/route/GuessNumberRoute.java

```diff
@@ -29,16 +36,19 @@ public GuessNumberRoute(UseCaseFactory useCaseFactory, JSONSerializer serializer
     public Object handle(Request request, Response response) {
         int id = extractIdParam(request);
         int guessNumber = extractAndValidateGuessNumber(request);
-
+        LOGGER.info("Request accepted. ID: " + id + " guessNumber: " + guessNumber);
         GuessNumberUseCase interactor = useCaseFactory.buildGuessNumberUseCase();
-        RestGuessResponseEntity result = new GuessResponseConverter().convert(interactor.checkGuessAndReturnResponse(id, guessNumber));
+        BoundaryGuessResponse boundaryGuessResponse = interactor.checkGuessAndReturnResponse(id, guessNumber);
```

Comment:
čia duosiu užuominą, kad galima daug gražiau su optional padaryt, pradedant nuo return Optional.of(interactor)...